src/apps/activity_assignments/domain/assignments.py

```diff
@@ -0,0 +1,37 @@
+from uuid import UUID
+
+from pydantic import BaseModel, root_validator
+
+from apps.activity_assignments.errors import ActivityAssignmentActivityOrFlowError
+from apps.shared.domain import InternalModel, PublicModel
+
+
+class ActivityAssignmentCreate(BaseModel):
+    activity_id: UUID | None
+    activity_flow_id: UUID | None
+    respondent_subject_id: UUID
+    target_subject_id: UUID
+
+    @root_validator
+    def validate_assignments(cls, values):
+        if not values.get("activity_id") and not values.get("activity_flow_id"):
```

Comment:
I was able to supply both `activity_id` and `activity_flow_id` and get a successful response:

![CleanShot 2024-08-06 at 16 57 43@2x](https://github.com/user-attachments/assets/2b92c375-b508-4f19-901b-582b715e37db)

```suggestion
        activity_id = values.get("activity_id")
        activity_flow_id = values.get("activity_flow_id")

        # Must supply either activity_id or activity_flow_id, but not both
        if (not activity_id and not activity_flow_id) or (activity_id and activity_flow_id):
```

Reply:
fixed and added tests to validate it